iceoryx_posh/include/iceoryx_posh/internal/popo/base_publisher.inl

```diff
@@ -1,4 +1,5 @@
-// Copyright (c) 2020 by Robert Bosch GmbH, Apex.AI Inc. All rights reserved.
+// Copyright (c) 2020 by Robert Bosch GmbH. All rights reserved.
+// Copyright (c) 2020 - 2021 by Apex.AI Inc. All rights reserved.
 //
 // Licensed under the Apache License, Version 2.0 (the "License");
 // you may not use this file except in compliance with the License.
```

Comment:
If this would be my PR I would reject this. This could have been trivially automated but since it's now already done on some files it's harder to do it.

Reply:
Would do this at most here for files with substantial changes, not for those with simple renaming. This just generates busy work which helps no one. So I will at least partially reject this, we need to find a better way.

Reply:
Please don't add the SPDX license identifier. There is a PR open to add them to all files.

Reply:
Does this include the whole header? Or only the last line?

Reply:
only the last line

Reply:
Done by script automatically I guess, newly created files should have this.